docs/changelog2.rst

```diff
@@ -0,0 +1,433 @@
+
+
+
+Changelog
+=========
+
+
+Version 2.0.0 (in progress)
+---------------------------
+
+Shapely 2.0 version is a major release featuring a complete refactor of the
+internals and new vectorized (element-wise) array operations providing
+considerable performance improvements (based on the developments in the
+`PyGEOS <https://github.com/pygeos/pygeos>`__ package), along with several
+breaking API changes and many feature improvements.
+
+For more background, see
+`RFC 1: Roadmap for Shapely 2.0 <https://github.com/shapely/shapely-rfc/pull/1>`__.
+
+
+Refactor of the internals
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Shapely wraps the GEOS C++ library for use in Python. Before 2.0, Shapely
+used ``ctypes`` to link to GEOS at runtime, but doing so resulted in extra
+overhead and installation challenges. With 2.0, the internals of Shapely have
+been refactored to expose GEOS functionality through a Python C extension
+module that is compiled in advance.
+
+The pointer to the actual GEOS Geometry object is stored in a lightweight
```

Comment:
My feeling is that it is too technical for most people to have this section directly here at the top. The information here is not relevant for most users.

Maybe better to move this paragraph in a separate section, after the "vectorization" part?

Reply:
Good point. 
Or alternatively, I could also move this text to some "design" page in the documentation (xref https://github.com/shapely/shapely/issues/1236) and link to that here.